iceoryx_posh/test/moduletests/test_mq_interface.cpp

```diff
@@ -228,6 +228,24 @@ void CMqInterface_StringCTor()
     EXPECT_THAT(base.getInterfaceName(), Eq(ifName));
 }
 
+template <typename T>
+void CMqInterface_close(T& base)
```

Comment:
This appears only to be a useful indirection when multiple types are tested and the verification takes more than one line or so... 
As it is, the indirection is just confusing and seems to have no benefit.

Is there any other intention I may have missed?
As it is now it would be more obvious and readable to call the corresponding EXPECTs directly in the test.

If the test cases are all to be run for different types TYPED_TESTs are the better choice. But this affects the file as a whole, the other tests also do not follow that principle...

Reply:
Yeah, as i remarked if we call EXPECTs directly in the test, this affects the file as a whole, the other tests also do not follow that principle, in this case, what do you think ? should i add EXPECT to only my tests, or all the file shoul be refactoring ?